### PR TITLE
removed properties from score spec

### PR DESCRIPTION
**Pull request by rachfop**
Thank you for submitting this pull request! We appreciate you spending the time to work on these changes.

## What is the motivation?

- https://github.com/score-spec/spec/issues/61
- https://github.com/score-spec/score-humanitec/pull/41
- https://github.com/score-spec/score-compose/pull/34
- https://github.com/score-spec/score-helm/pull/20

## What does this change do?

Provide a description of what this pull request does.

## What is your testing strategy?

Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.

## Is this related to any issues?

If this pull request is related to any other pull request or issue, or resolves any issues - then link all related pull requests and issues here.

## Have you read the [Contributing Guidelines](CONTRIBUTING.md)?

- [x] I have read the [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: content/en/docs/reference/score-spec-reference.md [hunk 1]
```diff
@@ -92,29 +86,16 @@ resources:
...
+| `volume`      | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification. | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification.                              |
+| `service`     | N/A                                                                                                                             | Translates to the module properties references. For example: `${modules.service-name.property-name}`.                                                        |
+| `workload`    | N/A                                                                                                                             | Reserved resource type. Its usage may lead to compatibility issues with future releases of [score-humanitec](https://github.com/score-spec/score-humanitec). |
 
 ### Referencing resources
```
Comment (sujaya-sys) [inline content/en/docs/reference/score-spec-reference.md:100]: This section should be updated as well, what do you think @ghen?

- Line 102: "Resources are declared in the containers section of the Score file and must map to the YAML structure defined in the resources section to properly resolve." - This sentence doesn't make total sense I think. Resources are declared in a dedicated `resources` section of the Score file.

- Line 107: "${resources.resource-name.property-name}" - This is no longer possible. Same for the example below.

File: content/en/docs/reference/score-spec-reference.md [hunk 2]
```diff
@@ -142,7 +123,7 @@ resources:
 
 It is up to the Score implementation (CLI) on how and when the resource reference is resolved, and when the referenced values' substitution occurs.
 
-For example, `score-compose` would convert resource properties into environment variables references in a resulting `compose.yaml` configuration file, and produce a reference `.env` file that the user can then populate. For more information, see the [.env file](https://docs.docker.com/compose/environment-variables/#the-env-file).
+For example, `score-compose` would convert resources into environment variables references in a resulting `compose.yaml` configuration file, and produce a reference `.env` file that the user can then populate. For more information, see the [.env file](https://docs.docker.com/compose/environment-variables/#the-env-file).
```
Comment (sujaya-sys) [inline content/en/docs/reference/score-spec-reference.md:126]: I'm not sure if this is correct, specifically referring to "`score-compose` would convert resources into environment variables" - @ghen what do you think? Would be great to discuss a more accurate wording

File: content/en/docs/reference/score-spec-reference.md (hunk 1 above)
Comment (ghen) [reply content/en/docs/reference/score-spec-reference.md:100]: Line 102 - Resources section in the container specification has nothing to do with the resources section in the score file. This whole sentence should be removed.

Comment (ghen) [reply content/en/docs/reference/score-spec-reference.md:100]: Line 110-113 - not sure what this block mean. We do not have it anywhere in the score file.

File: content/en/docs/get started/Transform/docker.md [hunk 3]
```diff
@@ -36,21 +36,6 @@ db:
...
-        default: 5432
-      name:
-      username:
-        secret: true
-      password:
-        secret: true
-```
-
 1. Declare your dependencies
 2. Add environment variables.
```
Comment (ghen) [inline content/en/docs/get started/Transform/docker.md:41]: What is the step 3 about? (see in this file bellow)

File: content/en/docs/reference/score-spec-reference.md [hunk 4]
```diff
@@ -92,57 +86,32 @@ resources:
...
-```yaml
-${resources.resource-name.property-name}
-```
-
-```yml
-resources:
-  resource-name:
-    property-name: my-property-name
-```
+Resources declared in the resources section of a Score file can be used in substitution patterns in different places
```
Comment (ghen) [inline content/en/docs/reference/score-spec-reference.md:102]: dot (`.`) is missing at the end of the sentence.

File: content/en/docs/reference/score-spec-reference.md [hunk 5]
```diff
@@ -92,57 +86,32 @@ resources:
...
 
 {{% alert %}}
 
-> If the referenced resource or its property has not been defined, the {{< glossary_tooltip text="Score implementation (CLI)" term_id="score" >}} should report a syntax error.
+> If the referenced resource or its property aren't defined, the {{< glossary_tooltip text="Score implementation (CLI)" term_id="score" >}} should report a syntax error.
```
Comment (ghen) [inline content/en/docs/reference/score-spec-reference.md:106]: This statement refers to properties that should be defined. And we do not declare properties anymore. Should remove any mention of the properties.

**Review by ghen — CHANGES_REQUESTED** [review]
Several fixes are needed.

File: content/en/docs/dependencies/_index.md [hunk 6]
```diff
@@ -30,17 +30,8 @@ containers:
...
-        type: string
-        default: World
   db:
     type: postgres
-    properties:
-      host:
-        default: localhost
-      port:
-        default: 5432
       name:
```
Comment (ghen) [inline content/en/docs/dependencies/_index.md:35]: Properties left overs. Should be removed.

File: content/en/docs/dependencies/dependencies-humanitec.md [hunk 7]
```diff
@@ -43,11 +43,6 @@ containers:
 resources:
   db:
     type: postgres
-    properties:
-      host:
-        default: localhost
-      port:
-        default: 5432
       name:
```
Comment (ghen) [inline content/en/docs/dependencies/dependencies-humanitec.md:46]: Properties leftover. Should be removed.

**Review by ghen — CHANGES_REQUESTED** [review]
Still some leftovers in YAML that should be removed.

**Review by ghen — APPROVED** [review]

**Pull request merged**